### PR TITLE
Added download file method to virtual machine

### PR DESCRIPTION
**Pull request by Steven17D**

### CODE REVIEW

File: vmwc/__init__.py [hunk 1]
```diff
@@ -663,6 +663,27 @@ def send_file(self, source_path, guest_path):
...
+                                                                                                    guestFilePath=guest_path)
+
+        resp = requests.get(fti.url.replace("https://*", "https://" + self._client.host), verify=False)
+
+        if not resp.status_code == 200:
+            raise Exception("Error while downloading file")
+
+        size = 0
+        with open(local_path, 'wb') as f:
+            for chunk in resp.iter_content(chunk_size=1024):
```
Comment (jossef) [inline vmwc/__init__.py]: the `1024` can be placed as an optional parameter so the user may change it when required;

```
def download_file(self, guest_path, local_path, chunk_size=1024)
```

File: vmwc/__init__.py [hunk 2]
```diff
@@ -663,6 +663,27 @@ def send_file(self, source_path, guest_path):
...
 
+    def download_file(self, guest_path, local_path):
+        if self._tools_credentials is None:
+            raise Exception("Login required to use guest functions")
+
+        fti = self._client.get_guest_operations_manager().fileManager.InitiateFileTransferFromGuest(vm=self._raw_virtual_machine,
+                                                                                                    auth=self._tools_credentials,
+                                                                                                    guestFilePath=guest_path)
+
+        resp = requests.get(fti.url.replace("https://*", "https://" + self._client.host), verify=False)
```
Comment (jossef) [inline vmwc/__init__.py]: you can use `self._client.verify` to determine if SSL verification is required in this session

File: examples/virtual-machine-download-file.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        for vm in client.get_virtual_machines():
+            print 'downloading "{}" from {}...'.format(remote_file_path, vm.name)
+            try:
+                size = vm.download_file(remote_file_path, download_path)
+                print 'successfully downloaded {} bytes'.format(size)
+            except Exception as e:
+                'Failed to download file: {}'.format(e)
+
+if __name__ == '__main__':
+    main()
```
Comment (jossef) [inline examples/virtual-machine-download-file.py]: great example, add it to the `README.md` file as well 
😉

**Review by jossef — CHANGES_REQUESTED** [review]
:1st_place_medal:

**Comment by Steven17D** [comment]
Hi Jossef, I fixed all the notes that you gave me.

**Review by jossef — APPROVED** [review]

**Pull request merged**